code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm

```diff
@@ -0,0 +1,212 @@
+// Category 2 medicines are medicines that have an ill effect regardless of volume/OD to dissuade doping. Mostly used as emergency chemicals OR to convert damage (and heal a bit in the process). The type is used to prompt borgs that the medicine is harmful.
+/datum/reagent/medicine/C2
+	harmful = TRUE
+
+/******BRUTE******/
+
+/datum/reagent/medicine/C2/arnica
+	name = "Arnica"
+	description = "A unique medicine that heals bruises, scaling with the rate at which one is bleeding out. Constricts blood streams, increasing the amount of blood lost. Overdosing further increases blood loss."
+	color = "#ECEC8D" // rgb: 236	236	141
+	taste_description = "wolf's bane"
+	overdose_threshold = 35
+	reagent_state = SOLID
+
+/datum/reagent/medicine/C2/arnica/on_mob_life(mob/living/carbon/M)
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		if(H.bleed_rate)
+			H.adjustBruteLoss(-0.5*H.bleed_rate*REM)
+			H.bleed_rate += 2
+	..()
+	return TRUE
+
+/datum/reagent/medicine/C2/arnica/overdose_process(mob/living/carbon/M)
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		H.bleed_rate += 2
+	..()
+	return TRUE
+
+/datum/reagent/medicine/C2/acetaminophen
+	name = "Acetaminophen"
+	description = "A common pain reliever. Does minor liver damage."
+	color = "#ECEC8D" // rgb: 236	236	141
+	taste_description = "a common spacehold drug"
+	reagent_state = SOLID
+
+/datum/reagent/medicine/C2/acetaminophen/on_mob_life(mob/living/carbon/M)
+	M.adjustLiverLoss(1*REM)
+	M.adjustBruteLoss(-1*REM)
+	..()
+	return TRUE
+
+/******BURN******/
+
+/datum/reagent/medicine/C2/silver_sulfadiazine
+	name = "Silver Sulfadiazine"
+	description = "Used to treat serious burns. Scales with prolonged exposure, but causes burns to itch as well."
+	reagent_state = LIQUID
+	color = "#C8A5DC"
+	var/resetting_probability = 0
+	var/spammer = 0
+
+/datum/reagent/medicine/C2/silver_sulfadiazine/on_mob_life(mob/living/carbon/M)
+	M.adjustFireLoss(-0.25*current_cycle*REM)
+	if(prob(resetting_probability))
+		if(spammer < world.time)
+			to_chat(M,"<span class='warning'>You can't help but to itch the burn.</span>")
+			spammer = world.time + (10 SECONDS)
+		var/scab = rand(1,7)
+		M.adjustBruteLoss(scab*REM)
+		M.bleed(scab)
+		resetting_probability = 0
+	resetting_probability += (5*(current_cycle/10)) // 10 iterations = >51% to itch
+	..()
+	return TRUE
+
+/datum/reagent/medicine/C2/neomycin_sulfate
+	name = "Neomycin Sulfate"
+	description = "Used to treat burns. Does minor eye damage."
+	reagent_state = LIQUID
+	color = "#C8A5DC"
+	var/resetting_probability = 0
+	var/message_cd = 0
+
+/datum/reagent/medicine/C2/neomycin_sulfate/on_mob_life(mob/living/carbon/M)
+	var/obj/item/organ/eyes/the_hills_have = M.getorganslot(ORGAN_SLOT_EYES)
+	M.adjustFireLoss(-1*REM)
+	the_hills_have?.applyOrganDamage(1)
```

Comment:
What the _fuck_ is this variable name. Also, considering eye damage will be interesting to treat if eye treatment chemicals also succumb to this system, I feel this will probably be used significantly less often than the scaling silver sulfadiazine.

Reply:
the hills have (eyes)